tests/unit/admin/test-class-wc-rest-payments-orders-controller.php

```diff
@@ -437,6 +437,38 @@ public function test_create_customer_from_order_non_guest_with_customer_id() {
 		$this->assertSame( 'cus_exist', $result_order->get_meta( '_stripe_customer_id' ) );
 	}
 
+	public function test_create_customer_from_order_with_invalid_status() {
+		$order = WC_Helper_Order::create_order();
+		$order->set_status( 'completed' );
+		$order->save();
+
+		$this->mock_customer_service
+			->expects( $this->never() )
+			->method( 'get_customer_id_by_user_id' );
+
+		$this->mock_customer_service
+			->expects( $this->never() )
+			->method( 'create_customer_for_user' );
+
+		$this->mock_customer_service
+			->expects( $this->never() )
+			->method( 'update_customer_for_user' );
+
+		$request = new WP_REST_Request( 'POST' );
+		$request->set_body_params(
+			[
+				'order_id' => $order->get_id(),
+			]
+		);
+
+		$response = $this->controller->create_customer( $request );
+		$this->assertInstanceOf( 'WP_Error', $response );
```

Comment:
We can use `WP_Error::class` here instead of `'WP_Error'`.

Reply:
Good suggestion. Addressed with https://github.com/Automattic/woocommerce-payments/pull/2626/commits/b90c2279cb88a3f1bdb479da5cfe1c7712fdff66.